lib/arel_extensions/visitors/postgresql.rb

```diff
@@ -8,9 +8,9 @@ class Arel::Visitors::PostgreSQL
 
       DATE_FORMAT_DIRECTIVES = {
         '%Y' => 'YYYY', '%C' => 'CC', '%y' => 'YY',
-        '%m' => 'MM', '%B' => 'FMMonth', '%^B' => 'FMMONTH', '%b' => 'Mon', '%^b' => 'MON',
```

Comment:
Shouldn't this be squashed into the previous commit?

Reply:
It does